python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -116,10 +116,10 @@ def __init__(
             raise DagsterDbtProjectNotFoundError(f"project_dir {project_dir} does not exist.")
 
         packaged_project_dir = Path(packaged_project_dir) if packaged_project_dir else None
-        if using_dagster_dev() and packaged_project_dir:
+
+        current_project_dir = project_dir
+        if packaged_project_dir and packaged_project_dir.exists():
             current_project_dir = packaged_project_dir
-        else:
-            current_project_dir = project_dir
 
         manifest_path = current_project_dir.joinpath(target_dir, "manifest.json")
 
```

Comment:
`project_dir=project_dir,` may need to get updated to `current_project_dir` - the package data stuff isn't really under test until this PR